spec/directConnectConf.js

```diff
@@ -5,11 +5,16 @@ exports.config = {
   directConnect: true,
 
   framework: 'jasmine',
+
+  // Disabled until https://github.com/angular/protractor/issues/4253 is resolved
+  /*
   multiCapabilities: [{
     'browserName': 'chrome'
   }, {
     'browserName': 'firefox',
   }],
+  */
+  browserName: 'chrome',
```

Comment:
`browserName` should be set in `capabilities`. This should not matter since chrome is the default value if no capabilities are set.
```
capabilities: {
  browserName: 'chrome'
}
```